build/build.xml

```diff
@@ -277,6 +277,43 @@
 			</else>
 		</if>
 
+		<!-- Checkout latest viglink to ext folder -->
+		<available file="${dir}/ext" type="dir" property="add-viglink-ext" />
```

Comment:
No, this check just makes sure we don't try to add an extensions to an old version that does not have an extension directory. VigLink gets added to the folders after this.